src/backend/index.ts

```diff
@@ -148,9 +148,504 @@ ltijs.app.put('/entity', async (req, res) => {
   return res.send('Success')
 })
 
+// Provide endpoint to start embed flow on edu-sharing
+// Called when user clicks on "embed content from edusharing"
+ltijs.app.get('/edusharing-embed/start', async (_, res) => {
```

Comment:
as proposed yesterday I'd prefer the edusharing routes in a different file / files as route handlers or another more express way :)